resources/views/user/pages/account/profile.blade.php

```diff
@@ -145,14 +94,13 @@ class="form-row form-row form-row-last validate-required">
                                                                             <label for="">Address</label>
                                                                             <p id="billing_address_1_field"
                                                                                 class="form-row form-row form-row-wide address-field validate-required">
-                                                                                <input type="text" value=""
-                                                                                    placeholder="Address" name="address"
+                                                                                <input type="text" value="{{Auth::user()->address}}" name="address"
                                                                                     class="input-text">
                                                                             </p>
                                                                         </div>
                                                                         <div class="col-xs-12 col-sm-12">
                                                                             <input type="submit" class="btn btn-default"
-                                                                                name="Login" value="Update">
+                                                                                name="Update" value="Update">
```

Comment:
name để chữ thường đi e 
```suggestion
                                                                                name="update" value="Update">
```